cmd/upgrade/cluster/cmd.go

```diff
@@ -215,7 +217,7 @@ func runWithRuntime(r *rosa.Runtime, cmd *cobra.Command) error {
 	}
 
 	// Check mandatory parameters and enable interactive mode if needed
-	if currentUpgradeScheduling.ScheduleDate == "" && currentUpgradeScheduling.ScheduleTime == "" &&
+	if (currentUpgradeScheduling.ScheduleDate == "" || currentUpgradeScheduling.ScheduleTime == "") &&
```

Comment:
Is this intentional? So, user needs to provide both "ScheduleDate" and "ScheduleTime" to not jump to the interactive mode?

Reply:
updated to cover this case